### ISSUE TITLE
Dear author, I transformed your code to tensorflow, and I just got something confusing. Hoping for your help

### ISSUE THREAD

**Issue by YangWangsky** [pouya-bashivan/EEGLearn#23]
I used the sample data and your preprocessing, and I got almost 95% accuracy for CNN mode with the same architecture. 
And what confused me more is that the accuracy on train set, validation set and  test set was the same. 
I have checked several times. So maybe is there something wrong with the sample data?

**Comment by YangWangsky** [comment]
Emmm, sorry, there was something wrong with my code.

**Comment by pbashivan** [comment]
I thought I had replied to this issue but good to know you found the error.
